### ISSUE TITLE
Theme Settings per instance are not preserved in iPad

### ISSUE THREAD

**Issue by z31r4m** [DagAgren/toot#333]
Minor UI Bug on iPad(s) only: 

PROBLEM: Theme settings are not preserved on iPad. On iPhone I can have different theme settings per instance and switch quite easily. On iPad when I switch to another instance and switch the theme it acts GLOBALLY and switches/clobbers theme settings in the other instances.

STEPS:
1 Go to instance A and set Toot Theme.
2 Go to Instance B and set Obsidian
3 Now return to Instance A.

RESULT: Uh oh, why does A show Obsidian?
Basically the last theme you set for any number of instances will render in all instances.

EXPECTED: Switches to Toot Theme, as it does on iPhone. If I have 3 difference instances I can do it on iPhone. Not on iPad. 

Found in version previous to 1.8.2 build84, regressed on 
iPhone 8 plus, 
iPad Pros 10.5 + 12.9 2nd gen

**Comment by DagAgren** [comment]
This is by design. It is very hard to get behaviour that makes much sense if the different instances in the list have different themes.

**Comment by z31r4m** [comment]
I’ve moved to a competing app that actually does this on both iPhone and iPad so it can’t be that hard. The idea came from Twitter clients that also do this per account.
